tests/models/test_NaiveBayes.py

```diff
@@ -0,0 +1,135 @@
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+import pandas as pd
+from pymc3 import summary
+import scipy.stats
+from sklearn.model_selection import train_test_split
+
+from pymc3_models import GaussianNaiveBayes
+from pymc3_models.exc import PyMC3ModelsError
+
+
+class GaussianNaiveBayesTestCase(unittest.TestCase):
+    def setUp(self):
+        """
+        Set up a test case with synthetic data.
+        """
+
+        self.num_cats = 3
+        self.num_pred = 10
+        self.num_samples = 100000
+
+        # Generate data
+        ## Priors
+        self.alpha = np.ones(self.num_cats)
+        self.pi = np.random.dirichlet(self.alpha)
```

Comment:
I just set up Travis CI to run the unittests and realized I should have been setting the random seed. Could you do that in this file? Like here, https://github.com/parsing-science/pymc3_models/blob/master/tests/models/test_LinearRegression.py#L23.